dali/ft/filecopy.cpp

```diff
@@ -3136,12 +3131,17 @@ void FileSprayer::updateTargetProperties()
                      ((stricmp(aname,FArecordCount)==0)&&!gotrc) ||
                      ((stricmp(aname,"@blockCompressed")==0)&&copyCompressed) ||
                      ((stricmp(aname,"@rowCompressed")==0)&&copyCompressed)||
-                     (stricmp(aname,"@local")==0)
+                     (stricmp(aname,"@local")==0)||
+                     (stricmp(aname,"@checkSum")==0)||
```

Comment:
Is it correct to copy @checkSum?

could (does it need to) the calculated check sum as a result of spray ?